services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: str,
+    movie_id: int,
+    cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    movie_session = MovieSession.objects.all()
+    if session_date:
+        movie_session = movie_session.filter(
+            show_time__date=session_date
+        )
+
+    return movie_session
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+    session_id: int,
+    show_time: str = None,
+    movie_id: int = None,
+    cinema_hall_id: int = None
+) -> None:
+    movie_session = MovieSession.objects.get(id=session_id)

```

Comment:
Can you refactor this function so that you will not need to get movie session if none of the update parameters has been passed?